Sources/AppcuesKit/Data/Models/Experience.swift

```diff
@@ -173,7 +173,15 @@ extension Experience: Decodable {
         steps = try container.decode([Step].self, forKey: .steps)
         redirectURL = try? container.decode(URL.self, forKey: .redirectURL)
         nextContentID = try? container.decode(String.self, forKey: .nextContentID)
-        renderContext = .modal
+
+        // Check for an experience-level embed trait
+        if #available(iOS 13.0, *),
+           let embedTrait = self.traits.first(where: { $0.type == AppcuesEmbeddedTrait.type }),
+           let config = embedTrait.configDecoder.decode(AppcuesEmbeddedTrait.Config.self) {
+            renderContext = .embed(frameID: config.frameID)
+        } else {
+            renderContext = .modal
+        }
```

Comment:
cross referencing [android side comment](https://github.com/appcues/appcues-android-sdk/pull/424#discussion_r1255856949) here, this is the expected lookup in traits

Reply:
Yep. It'd be nice to not have to dig through the traits to find the `frameID`, but it's the only safe way.